docs/encryption.md

```diff
@@ -1,60 +1,113 @@
 # Encryption
 
-## Encrypting and decrypting
+## A Note on Security
 
-If you don’t choose to encrypt your file when you run
-`jrnl` for the first time, you can encrypt
-your existing journal file or change its password using this:
+While `jrnl` follows best practices, total security is never possible in the
+real world. There are a number of ways that people can at least partially
+compromise your `jrnl` data. See the [Privacy and Security](./security.md) page
+for more information.
+
+## Dependencies
```

Comment:
These notes about installing cryptography should be part of the "manual decryption" section, so they don't get confused as dependencies for jrnl as a whole.